code/game/gamemodes/cult/runes.dm

```diff
@@ -316,6 +319,12 @@ var/list/teleport_runes = list()
 		to_chat(user, "<span class='cult'>You[moveuserlater ? "r vision blurs, and you suddenly appear somewhere else":" send everything above the rune away"].</span>")
 		if(moveuserlater)
 			user.forceMove(get_turf(actual_selected_rune))
+		user.apply_damage(6, BRUTE)
+		var/mob/living/carbon/human/H = user
+		if(user.z != T.z)
```

Comment:
My point is: there's a difference between a cult base on the mining or science outpost (which is discoverable/raidable) and one in deep space (which isn't typically findable, let alone attackable).

The code for teleport runes should distinguish between the two. The former should have a substantial penalty. The latter should have a *severe* penalty... or cults will just continue creating their bases in places like the white ship, where they're functionally invincible.